ext/pdo_oci/pdo_oci.stub.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+/** @generate-class-entries */
+
+/** @not-serializable */
```

Comment:
Nit: You could add a `@strict-properties` tag so that it already forbids dynamic properties